docs/commands/add.md

```diff
@@ -5,40 +5,64 @@ title: modular add
 
 # `modular add <packageName>`
 
-Adds a new package by creating a new workspace at `packages/<packageName>`,
-omitting the scope if the package is
+Adds a new package by creating a new package at the workspace located at
```

Comment:
Does it _always_ use `packages` (when --path is _not_ supplied)? I'm thinking about the use case where someone has defined custom yarn `workspaces` values in their `package.json`

Reply:
Yep, we hardcode the package root to `packages` if no `--path` option is provided. Maybe it can be done better (for example, forcing the user to specify a `--path` if the `workspaces` entry doesn't contain `packages`), but I'd rather think of it when we support many package manager, because that's when it becomes complicated.